clients/js/package-lock.json

```diff
@@ -0,0 +1,6308 @@
+{
```

Comment:
Lets stick with yarn.lock ? since it is also supported by NPM as well now.

Reply:
since we're using npm directly instead of yarn. no point keeping the yarn.lock file.